index.ts

```diff
@@ -295,7 +296,148 @@ const getFileCoverageTool: Tool = {
   },
 };
 
-// Handlers
+//PR endpoints
+
+const listRepositoryPullRequestsTool: Tool = {
+  name: 'codacy_list_repository_pull_requests',
+  description:
+    'List pull requests from a repository that the user has access to. You can search this endpoint for either last-updated (default), impact or merged',
+  inputSchema: {
+    type: 'object',
+    properties: {
+      provider: {
+        type: 'string',
+        description:
+          "Organization's git provider: GitHub (gh), GitLab (gl) or BitBucket (bb). Accepted values: gh, gl, bb.",
+      },
+      organization: {
+        type: 'string',
+        description: 'Organization name on the Git provider',
+      },
+      repository: {
+        type: 'string',
+        description: 'Repository name on the Git provider organization',
+      },
+      search: {
+        type: 'string',
+        description: 'Filter the results searching by this string.',
+      },
```

Comment:
I'd remove this one, since it's not actually a search. It's a terrible parameter, according to the docs: 
> You can search this endpoint for either last-updated (default), impact or merged